doc/releases/changelog-0.36.0.md

```diff
@@ -629,13 +794,15 @@
 * The `QNSPSAOptimizer` now properly handles differentiable parameters, resulting in being able to use it for more than one optimization step.
   [(#5439)](https://github.com/PennyLaneAI/pennylane/pull/5439)
 
-* The `QNode` interface now resets if an error occurs during execution.
+* The QNode interface now resets if an error occurs during execution.
   [(#5449)](https://github.com/PennyLaneAI/pennylane/pull/5449)
 
-* Fix failing tests due to changes with Lightning's adjoint diff pipeline.
+* Failing tests due to changes with Lightning's adjoint diff pipeline have been fixed.
   [(#5450)](https://github.com/PennyLaneAI/pennylane/pull/5450)
 
-* Fix Torch tensor locality with autoray-registered coerce method.
+Context: When Torch has a GPU backed data-buffer, failures can occur when attempting to make autoray-dispatched calls to Torch method with paired CPU data. In this case, for probabilities on the GPU, and eigenvalues on the host (read from the observables), failures appeared with qml.dot, and can be reproduced from:
```

Comment:
@isaacdevlugt what's going on here?

Reply:
Oh oops! I copy-pasted in some github PR comments from #5438 to help me re-word the entry

```suggestion
```